cpp/util/etcd.cc

```diff
@@ -980,7 +980,8 @@ list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
   list<EtcdClient::HostPortPair> ret;
   for (const auto& h : hosts) {
     vector<string> hp(util::split(h, ':'));
-    CHECK_EQ(2, hp.size()) << "Invalid host:port string: '" << h << "'";
+    CHECK_EQ(static_cast<size_t>(2), hp.size()) << "Invalid host:port string: '"
```

Comment:
It's guaranteed to be "pretty big"... Big enough to contain a pointer type, something like that? But then, the `size()` method might already return that.

I wouldn't worry about it either way.